go-controller/pkg/cni/helper_linux.go

```diff
@@ -428,40 +423,28 @@ func (pr *PodRequest) ConfigureInterface(podLister corev1listers.PodLister, kcli
 		}
 	}
 
-	// Only configure IPv6 specific stuff and wait for addresses to become usable
-	// if there are any IPv6 addresses to assign. v4 doesn't have the concept
-	// of tentative addresses so it doesn't need any of this.
-	haveV6 := false
-	for _, ip := range ifInfo.IPs {
-		if ip.IP.To4() == nil {
-			haveV6 = true
-			break
-		}
-	}
-	if haveV6 {
-		err = netns.Do(func(hostNS ns.NetNS) error {
-			// deny IPv6 neighbor solicitations
-			dadSysctlIface := fmt.Sprintf("/proc/sys/net/ipv6/conf/%s/dad_transmits", contIface.Name)
-			if _, err := os.Stat(dadSysctlIface); !os.IsNotExist(err) {
-				err = setSysctl(dadSysctlIface, 0)
-				if err != nil {
-					klog.Warningf("Failed to disable IPv6 DAD: %q", err)
-				}
+	err = netns.Do(func(hostNS ns.NetNS) error {
```

Comment:
@jcaamano Looks like this reverts the "don't settle v4 addresses" bits?

Reply:
Yup, fixed this.